src/main/asciidoc/index_en.adoc

```diff
@@ -214,27 +226,54 @@ Code of conduct: https://www.apache.org/foundation/policies/conduct.html
 image:fireworks.jpg[float=right]
 --
 
+== ServiceComb Incubating Journey
+
+image:ServiceCombIncubating.png[]
+
 == Who is Willem Jiang ?
 * Huawei OpenSource Center Techincal Expert
 * Former RedHat Principle Engineer
 * Apache Member
 * VP Apache ServiceComb
 
-== Apache License
-* Universal donor
-* Pragmatic
-* Free for commercial and non-commercial use
-* Compatible with other Open Source licenses
+== ServiceComb Introduction
+* **Java Chassis** - High performance Java service engine with MicroServices governance
+* **Service Center** - High performance Service Center based on Etcd
+* **Pack** - A Centrical distributed transaction coordinator which supports Saga and TCC
+* **Mesher** - A services mesh implementation based on Go
+* **Kie** - A new designed MicroServices configuration center
+
+== Find a way to Apache
+* Writing a proposal of your project
+* Finding a Champion and Mentors
+* Proposal Discussion in IPMC
+* Starting a vote of your proposal
+* Creating a podling project inside of Apache Incubator
+* SGA, CLA, iCLA Signing
+* TradeMark Searching
```

Comment:
replace with podling name search?

Reply:
sure, I will update the document for it.